user-interface/src/my-cases/MyCasesScreen.tsx

```diff
@@ -43,6 +46,10 @@ export const MyCasesScreen = () => {
     },
   };
 
+  useEffect(() => {
```

Comment:
I wonder if we can just do this in the `useLocationTracker` hook, rather than having to remember to add this `useEffect` boilerplate on every screen.

Reply:
Good idea.  We could do this with another hook that simply saves this info and doesn't return anything.